x-pack/plugins/siem/public/components/page/hosts/kpi_hosts/index.tsx

```diff
@@ -0,0 +1,194 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { EuiFlexGroup } from '@elastic/eui';
+import { get } from 'lodash/fp';
+import React from 'react';
+import { pure } from 'recompose';
+import { KpiHostsData } from '../../../../graphql/types';
+import {
+  AreaChartData,
+  BarChartData,
+  StatItem,
+  StatItems,
+  StatItemsComponent,
+  StatItemsProps,
+} from '../../../stat_items';
+import * as i18n from './translations';
+
+interface KpiHostsProps {
+  data: KpiHostsData;
+  loading: boolean;
+}
+
+const euiColorVis0 = '#00B3A4';
+const euiColorVis1 = '#3185FC';
+const euiColorVis2 = '#DB1374';
+const euiColorVis3 = '#490092';
+const euiColorVis9 = '#920000';
```

Comment:
Not sure if this is the 👍way to do it, but the following is used a bit around the codebase for importing the eui color constants:
``` ts
import euiDarkVars from '@elastic/eui/dist/eui_theme_dark.json';
import euiLightVars from '@elastic/eui/dist/eui_theme_light.json';
```

As for determining the current theme, there's the context api:

``` ts
<KibanaConfigContext.Consumer>
  {(config: Partial<AppKibanaFrameworkAdapter>) => {
    return config && config.darkMode ? euiDarkVars : euiLightVars;
  }}
</KibanaConfigContext.Consumer>
```

but I don't think we have a specific `ThemeProvider` for accessing color constants for the current theme outside what is wired up for use within `styled-components` (e.g. [empty.tsx](https://github.com/elastic/kibana/blob/feature-secops/x-pack/plugins/siem/public/components/timeline/data_providers/empty.tsx#L42)). That said, we may have some additional options when using the new elastic-charts.

Reply:
mm, thanks @spong that make sense! So far we haven't had kpis with different themes, and I couldn't find all the color codes here in existing configs. I've found`#490092`, `#920000`, should I replace them with import value? Still not sure what's preferable way of doing this.

Reply:
So in addition to importing the colors, this would be for ensuring the right color is chosen depending on which application-level theme is currently enabled -- either the Kibana Light or Dark Theme.

To import these, you could wrap the `KpiHostsComponent` with the `KibanaConfigContext.Consumer` example above, and then pass in the appropriate `euiDarkVars`/`euiLightVars` for whichever theme is selected. You would then have to restructure your component a bit and move `fieldTitleMapping` within `KpiHostsComponent` so you could use the appropriate colors (e.g. `euiDarkVars.euiColorVis3`).

I just doubled checked the [EUI colors](https://elastic.github.io/eui/#/guidelines/colors) used and it looks like they don't change between themes, so it's probably fine to leave things as they are. This is something we can re-visit when we update charting libraries.